README.md

```diff
@@ -14,8 +17,43 @@ The main goal is to inspire other teachers and share information about how to gi
     + [Javascript apis](#javascript-apis)
     + [JavaScript OOP: build a IMDB web scraper with real time code collaboration](#javascript-oop-build-a-imdb-web-scraper-with-real-time-code-collaboration)
     + [React - drawing components](#javascript-oop-build-a-imdb-web-scraper-with-real-time-code-collaboration)
-    
-    
+
+
+
+
+# General Teaching Techniques
+
+## Live Coding
+
```

Comment:
could you maybe add your name or the names of the teachers that help you draw up this info? In this way there is some sort of reference when people have questions about the tips that are described